### ISSUE TITLE
Erratic behavior of next/previous in "paused" state

### ISSUE THREAD

**Issue by tkem** [mopidy/mopidy#1177]
Given the following sequence of mpc commands
```
$ mpc load "Beatles - One"
loading: Beatles - One
$ mpc play
Love Me Do
[playing] #1/27   0:00/2:20 (0%)
volume:100%   repeat: off   random: off   single: off   consume: off
$ mpc pause
Love Me Do
[paused]  #1/27   0:06/2:20 (4%)
volume:100%   repeat: off   random: off   single: off   consume: off
$ mpc next
From Me to You
[paused]  #2/27   0:06/1:56 (5%)
volume:100%   repeat: off   random: off   single: off   consume: off
$ mpc play
From Me to You
[playing] #2/27   0:00/1:56 (0%)
volume:100%   repeat: off   random: off   single: off   consume: off
$ mpc status
From Me to You
[playing] #2/27   0:06/1:56 (5%)
volume:100%   repeat: off   random: off   single: off   consume: off
```
one would assume that the track that's getting played is "From Me to You". Curiously, however, it still is "Love Me Do", though `mpc status` claims otherwise.
This seems to occur only when changing tracks using `next`/`prev` in "paused" mode, and is also reproducible with Mopidy-Mobile/Mopidy.js, so it does not seem to be an `mpd` issue.
When calling `mpc stop` instead of `mpc pause`, this does not seem to occur, i.e. tracks are changed an played as expected, although `mpd status` will not report the current/next track when paused.

Mopidy v1.0.4 verbose log for above command sequence: https://gist.github.com/tkem/393e2f5c103b9eb238d5

**Comment by jodal** [comment]
I'm guessing this might have the same root cause as #1172.

**Comment by adamcik** [comment]
#939 comes to mind, but that was for seek but likely due to the same set of assumptions in the existing code.

**Comment by jodal** [comment]
Fixed with the merge of #1185.
